src/components/CourtDateFilter/CourtDateFilterOptions.tsx

```diff
@@ -4,10 +4,13 @@ import { NamedDateRangeOptions } from "utils/namedDateRange"
 import RadioButton from "components/RadioButton/RadioButton"
 import type { FilterAction } from "types/CourtCaseFilter"
 import type { Dispatch } from "react"
+import DateInput from "components/CustomDateInput/DateInput"
 
 interface Props {
   dateRange?: string | null
   dispatch: Dispatch<FilterAction>
+  customDateFrom: Date | null
+  customDateTo: Date | null
```

Comment:
Then we could delete `customDateFrom` and `customDateTo` arguments

Reply:
Agreed to leave as is to reduce complexity.

Reply:
Agreed to leave as is to reduce complexity.